packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ScatterPlotViewer/ScatterPlotViewer.stories.js

```diff
@@ -12,7 +12,7 @@ import {
   randomSingleSeriesData
 } from './helpers/mockData'
 import { Factory } from 'rosie'
-import readme from './README.md'
+// import readme from './README.md'
```

Comment:
This might need an import assertion in order to work properly. Maybe something along the lines of:

```js
import readme from './README.md' assert { type: "markdown" }
```

I'm still not sure what hidden magic is happening in the Storybook setup in order to enable Markdown imports.

Reply:
Maybe not. Node 18 only supports [JSON imports](https://nodejs.org/docs/latest-v18.x/api/esm.html#import-assertions).